contracts/access/AccessControlled.sol

```diff
@@ -2,20 +2,34 @@
 pragma solidity ^0.8.18;
 
 import { IZNSAccessController } from "./IZNSAccessController.sol";
-
+import { ZNSRoles } from "./ZNSRoles.sol";
 
 abstract contract AccessControlled {
 
     event AccessControllerSet(address accessController);
 
     IZNSAccessController internal accessController;
 
-
     modifier onlyAdmin() {
         accessController.checkAdmin(msg.sender);
         _;
     }
 
+    modifier onlyGovernor() {
```

Comment:
probably don't need this. It will overload the size of every contract. we have these on access controller already. should be enough